Zotero/Controllers/Database/Requests/CreateItemFromDetailDbRequest.swift

```diff
@@ -61,6 +62,7 @@ struct CreateItemFromDetailDbRequest: DbResponseRequest {
             guard let creator = self.data.creators[creatorId] else { continue }
 
             let rCreator = RCreator()
+            rCreator.uuid = UUID().uuidString
```

Comment:
It shouldn't really matter to be honest. Those `uuid`s are only used by UI to identify creators in current edit session. They don't sync to backend or matter otherwise at all. 

In this case we're duplicating existing item, so we're creating duplicates of `Creator`s too, which should probably have their own unique identifiers. `uuid` is not a primary key though, so even `creator.id` would still work.

Reply:
I've seen that in other places in the code they are kept in sync, but yes, if UI is not affected, that's ok.